logutils/stdhandler.go

```diff
@@ -0,0 +1,15 @@
+package logutils
+
+import (
+	stdlog "log"
+
+	"github.com/ethereum/go-ethereum/log"
+)
+
+// NewStdHandler returns handler that uses logger from golang std lib.
+func NewStdHandler(fmtr log.Format) log.Handler {
+	return log.FuncHandler(func(r *log.Record) error {
+		line := fmtr.Format(r)
+		return stdlog.Output(2, string(line))
```

Comment:
Let's explain `2`.

Reply:
tbh that was a default value used for calldepth in log.Println/Printf. and i don't completely understand why it is 2. will try to figure that out :)

Reply:
turned out ethereum.Handler adds a lot of additional frames. so if someone will want to print where call to a logger was made we will have to skip 8 frames. will add explainer to file